### DISCUSSION TITLE
RuntimeError: Given groups=1, weight of size [256, 1024, 1, 1], expected input[28, 2048, 15, 15] to have 1024 channels, but got 2048 channels instead

### DISCUSSION THREAD

**Discussion by Dtappledoghuati** [STVIR/pysot#328] [General]
My last question was
File "/media/misstian/tnq/pysot-master/tools/pysot/datasets/dataset.py", line 216, in _get_bbox
    imh, imw = image.shape[:2]
AttributeError: 'NoneType' object has no attribute 'shape'

When I adjusted somewhere the problem became
RuntimeError: Given groups=1, weight of size [256, 1024, 1, 1], expected input[28, 2048, 15, 15] to have 1024 channels, but got 2048 channels instead

I don't know if the previous problem is solved and what is going on

**Comment by ZhiyuanChen** [comment]
What exactly did you adjusted?

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now. 
Please create a new issue should you have other questions.
